fastai/optimizer.py

```diff
@@ -27,15 +39,27 @@ def freeze(self):
         assert(len(self.param_lists)>1)
         self.freeze_to(-1)
 
-    def set_freeze(self, n, rg, ignore_force_train=False):
+    def set_freeze(self,
+        n:int,
+        rg:bool, # Gradient flag for layer `n`
```

Comment:
I think just saying "Whether or not grad is required" or something like that would clarify this further.

Reply:
"Whether grad is required" should suffice

Reply:
Thx!